Komf no longer works on Kavita nightly

Updated Kavita to 0.8.6.9, and after that I can no longer match any series.
After looking up a series and selecting the correct one, I get this error and it never writes any metadata


```
Exception in thread "DefaultDispatcher-worker-3" io.ktor.serialization.JsonConvertException: Illegal input: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing at path: $

	at io.ktor.serialization.kotlinx.KotlinxSerializationConverter.deserialize(KotlinxSerializationConverter.kt:77)

	at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1$2.emit(Emitters.kt:51)

	at kotlinx.coroutines.flow.FlowKt__BuildersKt$asFlow$$inlined$unsafeFlow$3.collect(SafeCollector.common.kt:111)

	at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1.collect(SafeCollector.common.kt:109)

	at kotlinx.coroutines.flow.FlowKt__ReduceKt.firstOrNull(Reduce.kt:247)

	at kotlinx.coroutines.flow.FlowKt.firstOrNull(Unknown Source)

	at io.ktor.serialization.ContentConverterKt.deserialize(ContentConverter.kt:99)

	at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:234)

	at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.access$ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:1)

	at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invokeSuspend(ContentNegotiation.kt:249)

	at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)

	at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)

	at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invokeSuspend(KtorCallContexts.kt:105)

	at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)

	at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)

	at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)

	at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)

	at io.ktor.client.HttpClient$4.invokeSuspend(HttpClient.kt:1379)

	at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)

	at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)

	at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)

	at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)

	at io.ktor.client.plugins.ReceiveError$install$1.invokeSuspend(HttpCallValidator.kt:149)

	at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)

	at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)

	at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)

	at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)

	at io.ktor.util.pipeline.DebugPipelineContext.execute$ktor_utils(DebugPipelineContext.kt:63)

	at io.ktor.util.pipeline.Pipeline.execute(Pipeline.kt:79)

	at io.ktor.client.call.HttpClientCall.bodyNullable(HttpClientCall.kt:86)

	at snd.komf.mediaserver.kavita.KavitaClient.getSeries-JayA2N8(KavitaClient.kt:229)

	at snd.komf.mediaserver.kavita.KavitaClient$getSeries$1.invokeSuspend(KavitaClient.kt)

	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)

	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)

	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)

	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:829)

	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)

	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)

	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@2a0fdf74, Dispatchers.Default]
```

bump. also running into this with:
**Kavita Nightly**: 0.8.6.11 
**Komf**: 1.3.0
**komf-userscript**: 0.12.2
logs:


`22:33:05.204 [DefaultDispatcher-worker-2] ERROR snd.komf.mediaserver.metadata.MetadataService -- catching
io.ktor.serialization.JsonConvertException: Illegal input: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing at path: $
        at io.ktor.serialization.kotlinx.KotlinxSerializationConverter.deserialize(KotlinxSerializationConverter.kt:77)
        at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1$2.emit(Emitters.kt:51)
        at kotlinx.coroutines.flow.FlowKt__BuildersKt$asFlow$$inlined$unsafeFlow$3.collect(SafeCollector.common.kt:111)
        at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1.collect(SafeCollector.common.kt:109)
        at kotlinx.coroutines.flow.FlowKt__ReduceKt.firstOrNull(Reduce.kt:247)
        at kotlinx.coroutines.flow.FlowKt.firstOrNull(Unknown Source)
        at io.ktor.serialization.ContentConverterKt.deserialize(ContentConverter.kt:99)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:234)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.access$ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:1)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invokeSuspend(ContentNegotiation.kt:249)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invokeSuspend(KtorCallContexts.kt:105)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.client.HttpClient$4.invokeSuspend(HttpClient.kt:1379)
        at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)
        at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.client.plugins.ReceiveError$install$1.invokeSuspend(HttpCallValidator.kt:149)
        at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)
        at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.util.pipeline.DebugPipelineContext.execute$ktor_utils(DebugPipelineContext.kt:63)
        at io.ktor.util.pipeline.Pipeline.execute(Pipeline.kt:79)
        at io.ktor.client.call.HttpClientCall.bodyNullable(HttpClientCall.kt:86)
        at snd.komf.mediaserver.kavita.KavitaClient.getSeries-JayA2N8(KavitaClient.kt:229)
        at snd.komf.mediaserver.kavita.KavitaClient$getSeries$1.invokeSuspend(KavitaClient.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:829)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: kotlinx.serialization.MissingFieldException: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing at path: $
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
        at kotlinx.serialization.json.Json.decodeFromString(Json.kt:165)
        at io.ktor.serialization.kotlinx.KotlinxSerializationConverter.deserialize(KotlinxSerializationConverter.kt:69)
        ... 37 common frames omitted
Caused by: kotlinx.serialization.MissingFieldException: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing
        at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
        at snd.komf.mediaserver.kavita.model.KavitaSeries.<init>(KavitaSeries.kt:21)
        at snd.komf.mediaserver.kavita.model.KavitaSeries.<init>(KavitaSeries.kt)
        at snd.komf.mediaserver.kavita.model.KavitaSeries$$serializer.deserialize(KavitaSeries.kt:21)
        at snd.komf.mediaserver.kavita.model.KavitaSeries$$serializer.deserialize(KavitaSeries.kt:21)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        ... 39 common frames omitted`

BUMP!!
this is still happening, some version info and additional info;
versions:
**Kavita Nightly**: 0.8.6.11 
**Komf**: 1.3.0
**komf-userscript**: 0.12.2
logs:

`22:33:05.204 [DefaultDispatcher-worker-2] ERROR snd.komf.mediaserver.metadata.MetadataService -- catching
io.ktor.serialization.JsonConvertException: Illegal input: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing at path: $
        at io.ktor.serialization.kotlinx.KotlinxSerializationConverter.deserialize(KotlinxSerializationConverter.kt:77)
        at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1$2.emit(Emitters.kt:51)
        at kotlinx.coroutines.flow.FlowKt__BuildersKt$asFlow$$inlined$unsafeFlow$3.collect(SafeCollector.common.kt:111)
        at io.ktor.serialization.ContentConverterKt$deserialize$$inlined$map$1.collect(SafeCollector.common.kt:109)
        at kotlinx.coroutines.flow.FlowKt__ReduceKt.firstOrNull(Reduce.kt:247)
        at kotlinx.coroutines.flow.FlowKt.firstOrNull(Unknown Source)
        at io.ktor.serialization.ContentConverterKt.deserialize(ContentConverter.kt:99)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:234)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt.access$ContentNegotiation$lambda$13$convertResponse(ContentNegotiation.kt:1)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invokeSuspend(ContentNegotiation.kt:249)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)
        at io.ktor.client.plugins.contentnegotiation.ContentNegotiationKt$ContentNegotiation$2$2.invoke(ContentNegotiation.kt)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invokeSuspend(KtorCallContexts.kt:105)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)
        at io.ktor.client.plugins.api.TransformResponseBodyHook$install$1.invoke(KtorCallContexts.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.client.HttpClient$4.invokeSuspend(HttpClient.kt:1379)
        at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)
        at io.ktor.client.HttpClient$4.invoke(HttpClient.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.client.plugins.ReceiveError$install$1.invokeSuspend(HttpCallValidator.kt:149)
        at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)
        at io.ktor.client.plugins.ReceiveError$install$1.invoke(HttpCallValidator.kt)
        at io.ktor.util.pipeline.DebugPipelineContext.proceedLoop(DebugPipelineContext.kt:79)
        at io.ktor.util.pipeline.DebugPipelineContext.proceed(DebugPipelineContext.kt:57)
        at io.ktor.util.pipeline.DebugPipelineContext.execute$ktor_utils(DebugPipelineContext.kt:63)
        at io.ktor.util.pipeline.Pipeline.execute(Pipeline.kt:79)
        at io.ktor.client.call.HttpClientCall.bodyNullable(HttpClientCall.kt:86)
        at snd.komf.mediaserver.kavita.KavitaClient.getSeries-JayA2N8(KavitaClient.kt:229)
        at snd.komf.mediaserver.kavita.KavitaClient$getSeries$1.invokeSuspend(KavitaClient.kt)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:829)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
Caused by: kotlinx.serialization.MissingFieldException: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing at path: $
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:95)
        at kotlinx.serialization.json.Json.decodeFromString(Json.kt:165)
        at io.ktor.serialization.kotlinx.KotlinxSerializationConverter.deserialize(KotlinxSerializationConverter.kt:69)
        ... 37 common frames omitted
Caused by: kotlinx.serialization.MissingFieldException: Field 'nameLocked' is required for type with serial name 'snd.komf.mediaserver.kavita.model.KavitaSeries', but it was missing
        at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
        at snd.komf.mediaserver.kavita.model.KavitaSeries.<init>(KavitaSeries.kt:21)
        at snd.komf.mediaserver.kavita.model.KavitaSeries.<init>(KavitaSeries.kt)
        at snd.komf.mediaserver.kavita.model.KavitaSeries$$serializer.deserialize(KavitaSeries.kt:21)
        at snd.komf.mediaserver.kavita.model.KavitaSeries$$serializer.deserialize(KavitaSeries.kt:21)
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:69)
        ... 39 common frames omitted`

--- a/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/KavitaMediaServerClientAdapter.kt
+++ b/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/KavitaMediaServerClientAdapter.kt
@@ -112,11 +112,10 @@ class KavitaMediaServerClientAdapter(private val kavitaClient: KavitaClient) : M
         metadata: MediaServerSeriesMetadataUpdate
     ) {
         val localizedName = metadata.alternativeTitles?.find { it.language != null }
-        if (metadata.title != null || localizedName != null) {
+        if (metadata.titleSort != null || localizedName != null) {
             val series = kavitaClient.getSeries(seriesId.toKavitaSeriesId())
             kavitaClient.updateSeries(
                 series.toKavitaTitleUpdate(
-                    metadata.title?.name,
                     metadata.titleSort?.name,
                     localizedName?.name
                 )
@@ -305,7 +304,7 @@ private fun KavitaSeriesMetadata.toMediaServerSeriesMetadata(series: KavitaSerie
         links = webLinks?.split(",")?.map { WebLink(it, it) } ?: emptyList(),
 
         statusLock = publicationStatusLocked,
-        titleLock = series.nameLocked,
+        titleLock = false,
         titleSortLock = series.sortNameLocked,
         summaryLock = summaryLocked,
         readingDirectionLock = false,
@@ -477,13 +476,11 @@ private fun kavitaSeriesResetRequest(seriesId: KavitaSeriesId): KavitaSeriesMeta
     return KavitaSeriesMetadataUpdateRequest(metadata)
 }
 
-private fun KavitaSeries.toKavitaTitleUpdate(newName: String?, newSortName: String?, newLocalizedName: String?) =
+private fun KavitaSeries.toKavitaTitleUpdate(newSortName: String?, newLocalizedName: String?) =
     KavitaSeriesUpdateRequest(
         id = id,
-        name = newName?.trim() ?: name,
         sortName = newSortName?.trim() ?: sortName,
         localizedName = newLocalizedName?.trim() ?: localizedName,
-        nameLocked = nameLocked,
         sortNameLocked = sortNameLocked,
         localizedNameLocked = localizedNameLocked,
 
@@ -492,10 +489,8 @@ private fun KavitaSeries.toKavitaTitleUpdate(newName: String?, newSortName: Stri
 
 private fun KavitaSeries.toKavitaCoverResetRequest() = KavitaSeriesUpdateRequest(
     id = id,
-    name = name,
     localizedName = localizedName,
     sortName = sortName,
-    nameLocked = false,
     sortNameLocked = false,
     localizedNameLocked = false,
 

--- a/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/model/KavitaSeries.kt
+++ b/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/model/KavitaSeries.kt
@@ -27,14 +27,12 @@ class KavitaSeries(
     val originalName: String,
     val localizedName: String? = null,
     val sortName: String,
-    val summary: String? = null,
     val pages: Int,
     val format: Int,
     val created: LocalDateTime,
     val folderPath: String,
     val coverImageLocked: Boolean,
     val localizedNameLocked: Boolean,
-    val nameLocked: Boolean,
     val sortNameLocked: Boolean,
 )
 

--- a/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/model/request/KavitaSeriesUpdateRequest.kt
+++ b/komf-mediaserver/src/commonMain/kotlin/snd/komf/mediaserver/kavita/model/request/KavitaSeriesUpdateRequest.kt
@@ -6,11 +6,9 @@ import snd.komf.mediaserver.kavita.model.KavitaSeriesId
 @Serializable
 data class KavitaSeriesUpdateRequest(
     val id: KavitaSeriesId,
-    val name: String,
     val localizedName: String? = null,
     val sortName: String,
     val coverImageLocked: Boolean,
-    val nameLocked: Boolean,
     val sortNameLocked: Boolean,
     val localizedNameLocked: Boolean
 )

